src/apps/expenses/components/Expenses.js

```diff
@@ -94,6 +94,9 @@ class Expenses extends React.Component {
           .itemsList {
             position: relative;
           }
+          .itemsList :global(.item) {
```

Comment:
This doesn't need to be `:global` if the className is created in this component.

Reply:
good point